### ISSUE TITLE
Uncaught TypeError: Cannot read property '0' of undefined

### ISSUE THREAD

**Issue by zilbert97** [kiteco/atom-plugin#657]
I disabled and then re-enabled Kite in Atom, now anything I try and type throws up an error even when going back and disabling Kite.

1. ...
2. ...

**Atom**: 1.45.0 x64
**Electron**: 4.2.7
**OS**: Mac OS X 10.15.4
**Thrown From**: [kite](https://github.com/kiteco/atom-plugin) package 0.179.0

### Stack Trace

Uncaught TypeError: Cannot read property '0' of undefined

```
At /Applications/Atom.app/Contents/Resources/app/static/<embedded>:11

TypeError: Cannot read property '0' of undefined
    at e.exports.checkShouldSkipClosingBracket (/packages/kite/dist/main.js:31:321034)
    at /packages/kite/dist/main.js:31:320597)
    at Function.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:1214120)
    at Emitter.emit (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:1215561)
    at TextEditor.emitWillInsertTextEvent (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:58338)
    at TextEditor.insertText (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:15349)
    at TextEditor.e.(anonymous function) [as insertText] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:258222)
    at TextEditor.insertNewline (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:15857)
    at TextEditor.e.(anonymous function) [as insertNewline] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:258222)
    at TextEditor.editor:newline (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:306102)
    at /Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:307172
    at TextBuffer.transact (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:492268)
    at TextEditor.transact (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:24215)
    at HTMLElement.s.(anonymous function) (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:307115)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:349290)
    at KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:1254747)
    at KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:1250881)
    at WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:284496)
```

### Commands

```
     -4:05.5.0 core:backspace (input.hidden-input)
     -3:59.8.0 autocomplete-plus:confirm (input.hidden-input)
 27x -3:58.3.0 core:move-right (input.hidden-input)
  2x -3:56 core:move-left (input.hidden-input)
  7x -3:54 core:backspace (input.hidden-input)
     -3:07.1.0 core:save (input.hidden-input)
     -2:50.4.0 kite:package-settings (input.hidden-input)
     -1:44.4.0 core:cut (input.hidden-input)
  4x -1:42.4.0 editor:newline (input.hidden-input)
  3x -1:17 core:paste (input.hidden-input)
     -0:41.6.0 line-ending-selector:show (atom-text-editor.editor.is-focused)
     -0:37.7.0 encoding-selector:show (atom-text-editor.editor.is-focused)
  3x -0:34.7.0 core:paste (input.hidden-input)
  2x -0:26.5.0 core:undo (input.hidden-input)
     -0:20.3.0 core:save (input.hidden-input)
  2x -0:13.1.0 editor:newline (input.hidden-input)
```

### Non-Core Packages

```
kite 0.179.0 
```

**Comment by metalogical** [comment]
Thanks for the report. We're looking at this now.

**Comment by tonycheang** [comment]
Thanks for reporting! Is there any context of the file you could share, either the surrounding code or perhaps the completion suggested?

**Comment by metalogical** [comment]
Hi @zilbert97 

We've released a tentative fix: 0.180.0. Please let us know if you still experience the issue, and if so, context would be very useful to debug the issue.

**Comment by tonycheang** [comment]
Closing due to inactivity.
